### ISSUE TITLE
[Enhancement] Providing native Accessibility methods in Forms

### ISSUE THREAD

**Issue by Poli97** [xamarin/Xamarin.Forms#9441]
## Summary
In order to make it easier to develop Accessible application it will be useful to provide the native Accessibility API to all the Forms Views.

## API Changes
To achieve that now you have to create a Custom Renderer and subclass a native view, and then override all the Accessibility methods needed. But if you want to handle accessibility events on multiple different elements you have to create a Custom Renderer and subclass for each one, and this takes lot of time and code, as it is shown here https://github.com/xamarin/Xamarin.Forms/issues/9397#issuecomment-582219833
So it will be useful to provide the accessibility methods directly in Forms main View, so that every View (like Button, Image, Label...) inherit them. Or at least be able to override those methods directly in every Custom Renderer without the need to subclass a native view.

## Intended Use Case
This will be useful to develop more Accessible application in an easier way. An example should be to detect when an Image receive the Accessibility Focus by Talkback/Voiceover and reproduce an appropriate sound when that happens (e.g. a barking dog sound if it is the image of a dog). This can be useful, as in my case, for develop educational mathematical and grammatical applications for blind or visually impaired children.

**Comment by PureWeen** [comment]
This will most likely not be accepted for Xamarin.Forms but the .NET MAUI architecture makes this a lot simpler

https://docs.microsoft.com/en-us/dotnet/maui/user-interface/handlers/customize
